pre_commit_hooks/insert_license.py

```diff
@@ -56,6 +58,11 @@ def main(argv=None):
     parser.add_argument('--insert-license-after-regex', default="",
                         help="Insert license after line matching regex (ex: '^<\\?php$')")
     parser.add_argument('--remove-header', action='store_true')
+    parser.add_argument(
+        "--flexible-years",
```

Comment:
I'm not sure about the clarity of this new arg naming.
What do you think about `--insert-current-year`?
I think it could make clearer the fundamental role/impact of this parameter.

Reply:
I was thinking about naming it `--upgrade-years`, but then I figured out it we should support removing headers with stale years and the names `upgrade` and `insert` don't work. 
I'm open to other names if you still don't like the proposed one. I was also thinking about `lenient-years`, or `non-strict-years`, as opposed to the strict behavior we have now.

Reply:
`--use-current-year` ?

Reply:
Done